src/Symfony/Component/AssetMapper/CacheWarmer/AssetMapperCacheWarmer.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\CacheWarmer;
+
+use Symfony\Component\AssetMapper\AssetMapper;
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\AssetMapper\CompiledAssetMapperConfigReader;
+use Symfony\Component\AssetMapper\Event\PreAssetsCompileEvent;
+use Symfony\Component\AssetMapper\ImportMap\ImportMapGenerator;
+use Symfony\Component\AssetMapper\Path\PublicAssetsFilesystemInterface;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\HttpKernel\CacheWarmer\CacheWarmerInterface;
+
+/**
+ * @author Ryan Weaver <ryan@symfonycasts.com>
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+final class AssetMapperCacheWarmer implements CacheWarmerInterface
```

Comment:
This warmer seems to have a lot in common with the CompileCommand (logically 😅 ) ... should we not try to keep that code in one place only ?

Reply:
That would be better indeed! However, the command outputs a lot of useful info through console's OutputInterface (which the warmer doesn't have access to). We could create a service containing the logic, with an optional OutputInterface. Not sure it worth it given the warmer in pretty concise?